<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SocialMediaAnalyzer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #181825;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#sidebar {
	background-color: #1e1e2e;
	border-radius: 16px;
	color: #cdd6f4;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">QWidget {
	background-color: #1e1e2e;
	border-radius: 16;
	color: #cdd6f4;
}

QLineEdit, QPushButton, QProgressBar {
	background-color: #313244;
	border-radius: 8px;
}

QProgressBar:chunk {
	background-color: #89b4fa;
	border-radius: 8px;
}

QScrollBar {
    border: none;
    background: transparent;
}

QScrollBar::handle {
    background: #cdd6f4;
    min-width: 20px;
    min-height: 20px;
	border-radius: 4px;
}

QScrollBar::vertical {
    width: 8px;
}

QScrollBar::add-line {
    border: none;
    background: none;
}

QScrollBar::sub-line {
    border: none;
    background: none;
}

QLineEdit {
	padding-left: 8px;
}
</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>36</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="search_bar">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="search_btn">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="text">
                 <string>üîç</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="cmd_output">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>36</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="network_graph_widget" name="network_graph">
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: #1e1e2e;
	border-radius: 16px;
}
QPixmap {
	border-radius: 16px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>network_graph_widget</class>
   <extends>QLabel</extends>
   <header>network_graph_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
